packages/core/cache/src/Cache.js

```diff
@@ -75,7 +75,10 @@ export default class Cache {
       await this.fs.writeFile(blobPath, data);
       return key;
     } catch (err) {
-      logger.error(`Error writing to cache: ${err.message}`);
+      logger.error({
+        origin: '@parcel/cache',
+        message: `Error writing to cache: ${err.message}`
```

Comment:
could we make a factory function that returns loggers with pre-bound names, kind of like Python's stdlib `logging` module?

https://docs.python.org/3.7/library/logging.html#logging.getLogger

Reply:
Sure, although I'm not really sure how you'd figure out the origin for it. Or if we even need an origin at all

Reply:
The origin and the name of the logger would be the same, yeah? something like `createLogger(origin: string): Logger`

Reply:
Actually after looking through the code pluginOptions is initialised before any plugin is even loaded. So that one can't be used, so it would have to be another param or we'd have to create a pluginOptions instance for every plugin.

Not sure if this is something we should be doing